ports/atmel-samd/common-hal/alarm/__init__.c

```diff
@@ -103,64 +91,56 @@ mp_obj_t common_hal_alarm_create_wake_alarm(void) {
 STATIC void _setup_sleep_alarms(bool deep_sleep, size_t n_alarms, const mp_obj_t *alarms) {
     alarm_pin_pinalarm_set_alarms(deep_sleep, n_alarms, alarms);
     alarm_time_timealarm_set_alarms(deep_sleep, n_alarms, alarms);
-    fake_sleep = false;
 }
 
 mp_obj_t common_hal_alarm_light_sleep_until_alarms(size_t n_alarms, const mp_obj_t *alarms) {
     _setup_sleep_alarms(false, n_alarms, alarms);
     mp_obj_t wake_alarm = mp_const_none;
 
+    // This works but achieves same power consumption as time.sleep()
+    PM->SLEEPCFG.reg = PM_SLEEPCFG_SLEEPMODE_STANDBY;
+    while (PM->SLEEPCFG.bit.SLEEPMODE != PM_SLEEPCFG_SLEEPMODE_STANDBY_Val) {
+    }
+    // Even though RAMCFG_OFF, SYSRAM seems to be retained.  Probably
+    // because RTC keeps sleepwalking.  Anyway, STDBYCFG should be
+    // left intact as 0 to retain SYSRAM.
+    #if 0
```

Comment:
Please remove anything that is `#if 0`'d out. The git history has a copy if we need it.